### PR TITLE
Milestone-5 of  Bookstore-CMS Project

### PR DESCRIPTION
**Pull request by Samitti**
Hi, I have changed the design to the specification given and styled it. 
Kindly review my code, thanks.

### CODE REVIEW

File: src/components/App.css [hunk 1]
```diff
@@ -0,0 +1,230 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.appConatiner {
+  background-color: var(--pale-grey);
+}
+
```
Comment (christianotieno) [inline src/components/App.css:9]: ## Improve the Design 
- [ ] Improve the page layout to match that of the design given. Currently, what we have is a page\s content leaning on the left side of the site and is not as responsive as expected. The styling should be **EXACTLY** the same as the one provided in the designs ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/exactlySameAsDesign.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/))

|Required Design|Current Implemetation|
|:---:|:---:|
|bookstore-design|bookstore-implementation|

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @Samitti 👋 
Good job, you have done implementing the design of this project. Keep it up 👍 
We are almost done, just a few changes to work on before we proceed, 👇 
### Happy coding 💻 
#### Christian Otieno (CR)

**Comment by Samitti** [comment]
Hi, I have added media-query to reflect the same design as required, kindly review my code.

**Comment by Samitti** [comment]
Hi, I have added media-query to reflect the same design as required, kindly review my code.

File: src/components/App.css [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+.bookPannel {
+  width: 100%;
+  height: 130px;
+  margin-top: 37px;
+  padding: 25px 27px 26px 27px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
```
Comment (uche-inyama) [inline src/components/App.css]: I would suggest you make 
- width: 90%; 
- margin: 20px auto, **to center it and leave a space between each book component**

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

Hi @Samitti :wave: 

I can see you have centered the content of the page, like my colleague asked. However, if you observe, you will notice that the **book component** is not centered properly, it stretches all the way to the right; there should be a margin on the right of each book card.

I have left a comment below, suggesting how you could resolve it.

That's it, make the change, then submit another form for a code review

#### If you need clarifications reach out on slack. @ Inyama, Uchechukwu Henry

### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

**Comment by Samitti** [comment]
Hi, I have made the changes requested. Kindly review my code.

**Comment by Samitti** [comment]
Hi, I am sorry to say this but my milestone-2 is not reviewed until now, can you suggest to me also what to do to that milestone...

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Samitti, great job on this project :+1: You did a great job fixing the comments outline by the previous TSE's :smile: Awesome, however, the app is not responsive as expected on large devices. Kindly check the screenshots to understand the problem

- The first has to do with the user profile not showing completely and the other one has to do with the horizontal scroll
pr
hr

[Optional]
- Currently, our app is not mobile-friendly, it would be nice if we implement mobile responsive for it
- Live link is broken

Kindly fix this and submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Samitti** [comment]
Hi, I have made all the changes required, kindly review my code.

**Review by trekab — APPROVED** [review]
### Approved! :tada: 
Well done working on the suggested changes @Samitti! You may proceed to the final milestone. Please update your deployed version as well.

__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**